spk/duplicity/Makefile

```diff
@@ -4,7 +4,8 @@ SPK_REV = 1
 SPK_ICON = src/duplicity.png
 
 BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel
-BUILD_DEPENDS += cross/boto cross/netifaces cross/wrapt cross/$(SPK_NAME)
+BUILD_DEPENDS += cross/boto cross/msgpack-python cross/netifaces cross/rackspace-novaclient
```

Comment:
Yes, if you change the entry there to `python-wheel`, then you need to adjust the Python package as well. See what we did with uwsgi: https://github.com/SynoCommunity/spksrc/commit/0a3851f7255ace78020d49662ce18e9a910b7491 and https://github.com/SynoCommunity/spksrc/blob/master/spk/python/Makefile#L16.

There are other options too (set --system-site-packages when creating the virtualenv would be the main alternative), but I think I'd prefer the above option.